pkgs/top-level/all-packages.nix

```diff
@@ -21426,6 +21426,7 @@ with pkgs;
 
   ispc = callPackage ../development/compilers/ispc {
     inherit (llvmPackages) stdenv;
+    xcode = darwin.xcode_14;
```

Comment:
Went with xcode 14 simply because that's what the CI in ispc's repo was using

Reply:
Given the inconvenience of xcode having to be manually added to the store, I wonder if we should (try to) go with the default `darwin.xcode` to minimize annoyance as that one's more likely to be used/cached